source/WorkingPractices/TestSuites/lfric_core.rst

```diff
@@ -3,11 +3,17 @@
 Testing LFRic Core
 ==================
 
-LFRic testing is run with the following command from a working copy:
+.. note::
+
+    At the LFRic Apps vn2.0 release, the cylc7 LFRic Core suite was deprecated and the make test-suite functionality removed. Only the cylc8 suite is now maintained.
+
+LFRic testing is launched with Cylc8 rose-stem commands (as in eg. LFRic Apps):
 
 .. code-block::
 
-    make test-suite
+    export CYLC_VERSION=8
+    rose stem --group=developer
+    cylc play <NAME-OF-SUITE>
```

Comment:
Can we make this a bit clearer? rose stem can take an extra argument for `WORKFLOW_NAME`.  If defined, I think user can `cylc play WORKFLOW_NAME`.  In the absence of a defined workflow name, it's just the name of the working copy I believe?

Reply:
Sorry Yash, I've just merged this change in. 

This wording is used throughout our WP's for all the cylc 8 suites so perhaps a new PR to make them all clearer if you wanted?